### PR TITLE
Update link to BST project

### PR DESCRIPTION
**Pull request by char-adadev**
The purpose of this PR is to update the link to the BST project: Sorted Array to BST.

### CODE REVIEW

File: 03-Binary-Search-Trees/06-Bst-Checkpoint.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 
-| Logistics             | Details                                     |
-| --------------------- | ----------------------------------------    |
-| Project details       | https://github.com/Ada-C16/tree-practice    |
-| JS Version            | https://github.com/Ada-C16/tree-practice-js |
-| Should you fork this? | Yes (Check project details)                 |
-| Submission method     | Learn                                       |
+| Logistics             | Details                                           |
+| --------------------- | ----------------------------------------------    |
+| Project details       | https://github.com/ada-c17/Sorted-Array-to-BST    |
```
Comment (kp483) [inline 03-Binary-Search-Trees/06-Bst-Checkpoint.md:5]:
```suggestion
| Project details       | [https://github.com/ada-c17/Sorted-Array-to-BST] (https://github.com/ada-c17/Sorted-Array-to-BST)   |
```

**Comment by kp483** [comment]
Looks good!

Can you add time and space complexity questions? I put them as zero points and make the paragraph questions! Otherwise it requires you to have a specific answer you're looking for - or if you want to do a short answer question, you can use regex to make it accept all responses.

**Comment by char-adadev** [comment]
Thank you for taking a look! I added the space/time complexity questions. This should be good for another review.

**Review by kp483 — APPROVED** [review]
LGTM

**Pull request merged**